src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -17,7 +17,13 @@
   CHAT("chat", CHAT_TEAM, CHAT_GLOBAL, CHAT_ADMIN), // Changes the default chat channel
   DEATH(
       Arrays.asList("death", "dms"), DEATH_ALL, DEATH_OWN), // Changes which death messages are seen
-  PICKER("picker", PICKER_AUTO, PICKER_ON, PICKER_OFF); // Changes when the picker is displayed
+  PICKER("picker", PICKER_AUTO, PICKER_ON, PICKER_OFF), // Changes when the picker is displayed
+  JOIN_MESSAGES(
+      Arrays.asList("join-messages", "jms"),
```

Comment:
Similar with `death`, how about `join` and `jms`

Reply:
I took `jms` from `OCN/ProjectAres` repo, in case people were used to it.